src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsintegrationapi/models/Alert.kt

```diff
@@ -0,0 +1,16 @@
+package uk.gov.justice.digital.hmpps.hmppsintegrationapi.models
+
+import java.time.LocalDate
+
+data class Alert(
+  val offenderNo: String? = null,
+  val type: String? = null,
+  val typeDescription: String? = null,
+  val code: String? = null,
+  val codeDescription: String? = null,
+  val comment: String? = null,
+  val createDate: LocalDate? = null,
+  val expiryDate: LocalDate? = null,
+  val expired: Boolean? = null,
+  val active: Boolean? = null,
```

Comment:
I am happy this has captured the full scope of the data fields, and that 'Alert' has been removed from 'type', 'typeDescription', 'code' and 'codeDescription'

Reply:
however, "createDate" and "expiryDate" are unconvential namings.

Reply:
should be dateCreated as its always past tense, and dateExpired because dateExpires implies it might be populated when the alert has not yet expired, and this isn't the case in our data probing. This date appears to only be populated when an alert is manually expired by a person.